common/lsp/lsp-file-utils_test.cc

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "common/lsp/lsp-file-utils.h"
+
+#include "absl/strings/str_cat.h"
+#include "gtest/gtest.h"
+
+namespace verible {
+namespace lsp {
+
+std::string PathPrefix(const std::string &path) {
+#ifdef _WIN32
+  return absl::StrCat("c:/", path);
```

Comment:
Is this something that we can only test on Windows, because the `std::filesystem::path` behaves differently there ?

I suggest to use a different drive letter so that we don't accidentally rely on existence of the drive letter. So instead of `c:/` maybe `y:/`.

Reply:
Fixed it